.circleci/config.yml

```diff
@@ -173,6 +203,32 @@ workflows:
                       branches:
                           only: develop
 
+            - run-ui-tests-chrome:
+                requires:
+                    - deploy-site-to-test
+                context: tfn-fdbt-automation-test
+                filters:
+                      branches:
+                          only:  
+                              - develop
+            
+            - run-ui-tests-firefox:
+                requires:
+                    - run-ui-tests-chrome
```

Comment:
Do they need to be sequential or can we run them all at the same time?

Reply:
We can but I wondered if it was worth running them all if one fails, in terms of cost